bot/message_handler.js

```diff
@@ -0,0 +1,22 @@
+function handleMessage(msg) {
```

Comment:
ES5 function used here. Use something like this:

`const handleMessage = (msg) => {}`